.github/workflows/build.yaml

```diff
@@ -153,26 +155,90 @@ jobs:
             # Sign the APK index
             melange sign-index -f --signing-key ./wolfi-signing.rsa packages/${arch}/APKINDEX.tar.gz
 
-            # Only attempt to sign when *.apk's exist.
+            # Only attempt to sign and upload APKINDEX when *.apk's changed.
             apks=$(ls ./packages/${arch}/*.apk 2>/dev/null || true)
             if [ -n "$apks" ]; then
               melange sign --signing-key ./wolfi-signing.rsa ./packages/${arch}/*.apk
+
+              # Don't cache the APKINDEX.
+              gcloud --quiet storage cp \
+                  --cache-control=no-store \
+                  "./packages/${arch}/APKINDEX.tar.gz" "gs://wolfi-production-registry-destination/os/${arch}/"
+
+              gcloud --quiet storage cp \
+                  --cache-control=no-store \
+                  "./packages/${arch}/APKINDEX.json" "gs://wolfi-production-registry-destination/os/${arch}/"
```

Comment:
I think this was intentionally outside of the loop to minimize the time the indices might be inconsistent

Reply:
Do we upload the index before the APKs?  That doesn't seem like something you are changing, but it seems like a terrible idea 😅